weblate_web/templates/snippets/service.html

```diff
@@ -1,94 +1,161 @@
+                                <span>
+                                    {% with days_from_today=subscription.expires|days_diff_from_today %}
+                                    {% if days_from_today < 30 %}
+                                    {% if days_from_today < 2 and days_from_today >= 0 %}
+                                    <t class="pl-1 orange">{{ days_from_today}} {% trans "day remaining" %}</t>
+                                    {% elif days_from_today < 0  %}
+                                    <t class="pl-1 orange">Expired</t>
+                                    {% else %}
+                                    <t class="pl-1 orange">{{ days_from_today}} {% trans "days remaining" %}</t>
+                                    {% endif %}
+                                    {% else %}
+                                    <t class="pl-1 grey">{{ days_from_today}} {% trans "days remaining" %}</t>
+                                    {% endif %}
+                                    {% endwith %}
+                                </span>
+                            </t>
+                            <br>
+                            <v class="subscription-note">Note:
+                                <span>
+                                    {% if service.note %}
+                                    {{ service.note }}
+                                    {% endif %}
+
+                                </span>
+                            </v>
+
                             {% if service.status == "shared" %}
                             <a class="button small inline" href="https://hosted.weblate.org/create/billing/?billing={{ service.hosted_billing }}&amp;upgrade=1">{% trans "Change plan" %}</a>
                             {% endif %}
                         </div>
                         <div class="clear"></div>
                     </div>
                     {% if subscription.get_repeat %}
-                    <div class="form-line">
+                    <div class="form-line style6">
                         <div class="line-left">{% trans "Validity" %}</div>
-                        <div class="line-right">{% blocktrans with created=subscription.created|date:"SHORT_DATE_FORMAT" expires=subscription.expires|date:"SHORT_DATE_FORMAT" context "Date range" %}{{ created }}–{{ expires }}{% endblocktrans %}</div>
-                        <div class="clear"></div>
-                    </div>
-                    {% else %}
-                    <div class="form-line">
-                        <div class="line-left">{% trans "Creation date" %}</div>
-                        <div class="line-right">{{ subscription.created|date:"SHORT_DATE_FORMAT" }}</div>
+                        <div class="line-right">{% blocktrans with created=subscription.created|date:"d M Y" expires=subscription.expires|date:"d M Y" context "Date range" %}{{ created }}–{{ expires }}{% endblocktrans %}</div>
```

Comment:
The validity is now included above, please remove it here.

Reply:
I didn't get you. I have not added anything above related to it

Reply:
Yes, i have gotten rid of that now.